Use the OCI Helm Charts repository instead of the manually maintained Helm Chart repo on the website (#8791)
strimzi/strimzi-kafka-operator: documentation/modules/deploying/proc-deploy-cluster-operator-helm-chart.adoc

.Prerequisites

* The Helm client must be installed on a local machine.

.Procedure

. Install the Strimzi Cluster Operator using the Helm command line tool:
+
[source,shell]
----
helm install strimzi-cluster-operator oci://quay.io/strimzi-helm/strimzi-kafka-operator
----
+

> Alternatively, you can also install an exact version of the Cluster Operator or specify any changes to the default configuration as parameter values.

Reviewer: ```suggestion
Alternatively, you can use parameter values to install a specific version of the Cluster Operator or specify any changes to the default configuration.
```
